TestFoundation/TestCodable.swift

```diff
@@ -33,25 +33,28 @@ private func makePersonNameComponents(namePrefix: String? = nil,
     return result
 }
 
-func expectRoundTripEquality<T : Codable>(of value: T, encode: (T) throws -> Data, decode: (Data) throws -> T) where T : Equatable {
+func expectRoundTripEquality<T : Codable>(of value: T, encode: (T) throws -> Data, decode: (Data) throws -> T) -> Bool where T : Equatable  {
     let data: Data
     do {
         data = try encode(value)
     } catch {
-        fatalError("Unable to encode \(T.self) <\(value)>: \(error)")
+        XCTFail("Unable to encode \(T.self) <\(value)>: \(error)")
+        return false
     }
 
     let decoded: T
     do {
         decoded = try decode(data)
     } catch {
-        fatalError("Unable to decode \(T.self) <\(value)>: \(error)")
+        XCTFail("Unable to decode \(T.self) <\(value)>: \(error)")
```

Comment:
Should this just be returning NO and leaving the XCTFail for the higher level call?

Reply:
I left it in because it acts as a handy debug message in the failure case and isn't any noiseier if the `decode` works.